Dart/thirdPartySrc/analysisServer/com/google/dart/server/LSPDartTextDocumentContentConsumer.java

```diff
@@ -0,0 +1,16 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package com.google.dart.server;
+
+import org.dartlang.analysis.server.protocol.RequestError;
+
+public interface LSPDartTextDocumentContentConsumer extends Consumer {
```

Comment:
Are `LSPDartTextDocumentContentConsumer` and `LSPDartTextDocumentContentProcessor` names generated?

If they are hand-written, are you ok with `DartLspTextDocumentContentConsumer` and `DartLspTextDocumentContentProcessor`? Not a big deal if not.

Reply:
Done.